config/config.js

```diff
@@ -4,7 +4,7 @@ const _ = require("lodash");
 const fs = require("fs");
 const path = require("path");
 
-const envConfig = process.env.NODE_ENV === "DEVTOOLS_PANEL" ?
+const envConfig = process.env.NODE_ENV === "production" ?
```

Comment:
I don't want to conflate the pane with production because dev/prod mode is something all libs use to turn on prod mode, and I'm sure there will be times where we want to do `NODE_ENV=production npm start` to try it in prod mode locally.